sunflower/gui/main_window.py

```diff
@@ -2480,6 +2480,7 @@ def register_class(self, name, title, PluginClass):
 
 		# create action
 		action = Gio.SimpleAction.new(name, None)
+		action.plugin_class = PluginClass
```

Comment:
Can you be more descriptive with changes? Why was this done and how it resulted in a fix?

Reply:
This fixed a crash at (IIRC) https://github.com/MeanEYE/Sunflower/blob/1aeeaff6439759867adb6d09a0094daa99429947/sunflower/gui/main_window.py#L924) – I don't know if that was Python 3 specific

Reply:
Okay, thanks. Makes sense.